src/main/kotlin/de/bund/digitalservice/useid/widget/WidgetController.kt

```diff
@@ -1,31 +1,54 @@
 package de.bund.digitalservice.useid.widget
 
 import de.bund.digitalservice.useid.config.ApplicationProperties
+import de.bund.digitalservice.useid.tracking.TrackingService
+import de.bund.digitalservice.useid.tracking.WidgetTracking
 import io.micrometer.core.annotation.Timed
 import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
 import org.springframework.http.server.reactive.ServerHttpRequest
 import org.springframework.stereotype.Controller
 import org.springframework.ui.Model
 import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.reactive.result.view.Rendering
 
 internal const val WIDGET_PAGE = "widget"
 internal const val INCOMPATIBLE_PAGE = "incompatible"
 internal const val FALLBACK_PAGE = "eID-Client"
+internal const val APP_OPENED = "app-opened"
```

Comment:
I know I suggested that name, but having a second thought, maybe "button clicked" is better?

Reply:
I agree. `app-opened` is not very self explanatory without knowing / understanding the context of the widget.